frigate/video.py

```diff
@@ -251,8 +251,9 @@ def run(self):
                 }
             )
 
-        time.sleep(10)
-        while not self.stop_event.wait(10):
+        sleeptime = self.config.ffmpeg.timeout
```

Comment:
There's no reason to read from the config every time, should be set as self.healthcheck_interval in init and then read here.